artemis-jms/src/main/resources/META-INF/resources/index.html

```diff
@@ -0,0 +1,152 @@
+<!DOCTYPE html>
```

Comment:
Can you drop this page (while generated, we don't want it in the quickstart code).

Reply:
Sure, should I also drop the rest?